core/templates/dev/head/pages/teach.html

```diff
@@ -48,56 +48,51 @@
             Teach with Oppia
           </h2>
 
-          <div class="oppia-about-card-content-wide oppia-about-three-cols-container">
-            <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
-              <img ng-src="<[getStaticImageUrl('/general/creator-create-exp.png')]>">
-              <h3>Create an Exploration</h3>
-              <p style="margin-top: -10px;">
-                about a topic you care about.
-              </p>
-            </div>
-            <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
-              <img ng-src="<[getStaticImageUrl('/general/creator-pub-share.png')]>">
-              <h3>Publish &amp; Share</h3>
-              <p style="margin-top: -10px;">
-                your creations with the community.
-              </p>
-            </div>
-            <div class="oppia-about-three-cols-layout oppia-about-three-cols-feature">
-              <img ng-src="<[getStaticImageUrl('/general/creator-feedback.png')]>">
-              <h3>Earn feedback</h3>
-              <p style="margin-top: -10px;">
-                to improve your exploration.
-              </p>
-            </div>
+          <div style="text-align: center; width: 100%;">
+            <em>Create high-quality lessons that help students learn difficult topics.</em>
           </div>
 
-          <hr>
+          <p>
+            You might know a student who's struggling with a topic at school.
+            They really want to learn it, but the lesson goes too fast, or the
+            textbook isn't easy to understand &mdash; and it's hard for them to
+            get the support they need. And it's not just this student: other
```

Comment:
Nit: redundant 'And' beginning sentence ('and' used as conjunction in close proximity within the previous sentence).